index.js

```diff
@@ -1,45 +1,24 @@
 'use strict';
 
-const apolloClientDefaultPackages = [
-  'apollo-cache',
-  'apollo-cache-inmemory',
-  'apollo-client',
-  'apollo-link',
-  'apollo-link-context',
-  'apollo-link-http',
-  'graphql',
-  'graphql-tools',
-  'graphql-tag',
-];
-
 module.exports = {
   name: 'ember-apollo-client',
 
-  included() {
-    this._super.included.apply(this, arguments);
-
-    const host = this._findHost();
-    this.addonConfig = host.options['apollo'] || {};
-
-    this.import('vendor/-apollo-client-bundle.js');
-    this.import('vendor/-apollo-client-shims.js');
-  },
-
-  treeForVendor() {
-    const WebpackDependencyPlugin = require('./lib/webpack-dependency-plugin');
-    const {
-      include: userPackages = [],
-      exclude: excludedPackages = [],
-    } = this.addonConfig;
-
-    const includedPackages = apolloClientDefaultPackages.filter(
-      p => !excludedPackages.includes(p)
-    );
-
-    return new WebpackDependencyPlugin({
-      outputName: 'apollo-client',
-      expose: [...includedPackages, ...userPackages],
-    });
+  options: {
+    autoImport: {
+      webpack: {
+        module: {
+          rules: [
```

Comment:
Only needed for `0.13.1` & `0.13.2`.  This issue does not exist in `0.13.0` and `>= 14.0.0-rc.1`.  Verified configuration is safe for all versions of `graphql-js`.